### DISCUSSION TITLE
the vasp prgram spend too much time in slurm

### DISCUSSION THREAD

**Discussion by goodluck-xyz** [deepmodeling/dpgen#1052] [Q&A]
I try to run dpgen in sugon platform. I run the `dpgen init_bulk init.json machine.json` in slrum. The `00.place_ele`  spend too much time. In dpgen program, it spend 1 hour. However, I copy the  vasp  input file and run it in slurm. It spend 6 minutes. How to decrease the running time?

**Comment by AnguseZhang** [comment]
The computing efficiency is dependent on the machine computing capacity. The difference between different computing platforms is not strange. You should better select proper computing resources.
